### PR TITLE
Adds support for 64 bit IDs. Fixes #27

### LINKED ISSUE #27: Use node-uuid.v4()
As of now, the traceIds and spanIds are generated using the function,

```JS
Math.floor(Math.random() * largestRandom);
```

Can we move to UUID (v4) instead?

### PR DESCRIPTION
**Pull request by suryatech**
Uses the node-int64 library to overcome the largest number limitations in Javascript.

### CODE REVIEW

**Comment by wision** [comment]
@suryatech can you please fix those jslint errors? thanks

**Comment by suryatech** [comment]
@wision Anything else?

**Comment by wision** [comment]
Now I hope that @cyli wil merge it:)

File: lib/trace.js [hunk 1]
```diff
@@ -53,14 +61,14 @@
...
-   * @param {Hash} optionalIds A hash any or all or none of the following
+   * @param {Hash} options A hash any or all or none of the following
    *    params, for which values will be generated if not provided:
    *
-   *    @param {Number} traceId 64-bit integer identifying this trace
-   *    @param {Number} spanId 64-bit integer identifying this span
-   *    @param {Number} parentSpanId 64-bit integer identifying this trace's
-   *        parent span, or undefined.
-   *    @param {Array} _tracers Zero or more {Tracers}
+   * @param {Number} options.traceId 64-bit integer identifying this trace
```
Comment (cyli) [inline lib/trace.js]: Thank you for fixing these docstrings!

**Comment by cyli** [comment]
Hi @suryatech!  Thanks so much for your contribution!  Since `node-int64` is required now, should it be moved from `devDependencies` to `dependencies` in `package.json`?  If so, could that be done in this PR?

Also completely non-blocking comment - the indentation on a bunch of the new code are tabs and don't line up with the old indentation - no need to change it in this PR if you didn't want to, but for consistency would you mind terribly if I did so later?  (I am just going by [this coding standard](https://docs.npmjs.com/misc/coding-style) - please correct me if that is no longer relevant).

Thanks again!

**Comment by suryatech** [comment]
@cyli Anything else that is needed for this to be merged?

I fixed the indentation issue in one of the previous commits. Let me know if something is out of

**Comment by cyli** [comment]
@suryatech It looks great - thank you so much for your contribution!

**Pull request merged**